data/data/baremetal/main.tf

```diff
@@ -21,24 +21,14 @@ module "masters" {
   source = "./masters"
 
   ignition       = "${var.ignition_master}"
-  image_source   = "${var.master_configuration["image_source"]}"
-  image_checksum = "${var.master_configuration["image_checksum"]}"
-  root_gb        = "${var.master_configuration["root_gb"]}"
-  root_disk      = "${var.master_configuration["root_disk"]}"
-
-  master_0 = "${var.master_0}"
-  properties_0 = "${var.properties_0}"
-  root_device_0 = "${var.root_device_0}"
-  driver_info_0 = "${var.driver_info_0}"
-
-  master_1 = "${var.master_1}"
-  properties_1 = "${var.properties_1}"
-  root_device_1 = "${var.root_device_1}"
-  driver_info_1 = "${var.driver_info_1}"
-
-  master_2 = "${var.master_2}"
-  properties_2 = "${var.properties_2}"
-  root_device_2 = "${var.root_device_2}"
-  driver_info_2 = "${var.driver_info_2}"
+  image_source   = "${var.master_configuration["ImageSource"]}"
+  image_checksum = "${var.master_configuration["ImageChecksum"]}"
+  root_gb        = "${var.master_configuration["RootGb"]}"
+  root_disk      = "${var.master_configuration["RootDisk"]}"
```

Comment:
Install is failing since these don't get set (dev-scripts is using image_source, etc)

Reply:
Actually my install is still failing for some other reason, but you should be able to set the HCL names for these in `pkg/types/baremetal/platform.go`:

```go
type MasterConfiguration struct {
        ImageSource   string `hcl:"image_source" json:"image_source,omitempty"`
        ImageChecksum string `hcl:"image_checksum" json:"image_checksum,omitempty"`
        RootGb        string `hcl:"root_gb" json:"root_gb,omitempty"`
}
```